e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -875,6 +909,24 @@ describe("scenarios > embedding > full app", () => {
       cy.intercept("GET", "/api/table/*/query_metadata").as("getTableMetadata");
     });
 
+    it('should respect "entity_types" search parameter (EMB-228)', () => {
+      cy.log('test `entity_types=["table"]`');
+      startNewEmbeddingQuestion({
+        isMultiStageDataPicker: true,
+        searchParameters: { entity_types: "table" },
+      });
+      H.popover().within(() => {
+        /**
+         * When we're in table step, it means we don't show models, otherwise, we would have shown
+         * the bucket step which has "Raw Data" and "Models" options instead.
+         */
+        cy.findByText("Sample Database").should("be.visible");
+        cy.findByRole("heading", { name: "Orders" }).should("be.visible");
```

Comment:
Should we assert that "Orders model" is not visible as well? Or is that not likely because `isMultiStageDataPicker` ensures that only a single entity type will ever be shown at once?

Reply:
This is already expected. In multi-stage data picker, you'll be greeted with the bucket step (named in our code).

![image](https://github.com/user-attachments/assets/3c25eda8-61ce-4565-81ef-4baa376dae99)


That means if we're greeted with the table step. There's already no model to choose to begin with
![image](https://github.com/user-attachments/assets/c0400384-e1ab-4a2b-ae35-067dad34c98c)